### PR TITLE
Finalise the Release Notes 3.2.3

### LINKED ISSUE #2565: Release notes 3.2.3
We shouldn't forget the release notes ...

### LINKED ISSUE #2646: Change in World-object causes regressions
The correction for #2432 causes regressions in a commercial library (FlightDynamics); and in theory there could be other similar cases.

Basically if a library made their own world-object (by copying World from MSL and adding parts) and used it as "inner MyWorld world;" the "outer World world;" in MSL no longer matches the inner component due to the new parameter for animateGround.

I don't see that we have stated that making your own inner-objects is not fully supported.

One possibility would be to accept it and explain that it is not supported.

### LINKED ISSUE #2662: Clarify if broken backward compatibility is acceptable
Commit 941e361c69eec73dc60ef118782d08189d913266 for #2364 adds a new constant (C_default) to a partial class (PartialMedium), which - strictly seen - breaks the desired backward compatibility of MSL v3.2.3. This is a discussion issue, if we accept it nevertheless and document it in the release notes or, should revert it to not break compatibility.

@MartinOtter @HansOlsson @hubertus65 Please comment and advice.

### PR DESCRIPTION
**Pull request by AHaumer**
started to fill release notes 3.2.3

close #2565
close #2646 
close #2662

### CODE REVIEW

**Comment by AHaumer** [comment]
Remember: You'll get a lot of pull requests, since nearly all library officers have to cintribute to the release notes. Maybe we should keep the relese notes in one or more separate files, avoiding to touch the main package.mo

**Comment by dietmarw** [comment]
We can simply use this PR do finalise the release notes. By default all users with push rights (i.e., all library officers) will be able to update this/your PR.

**Comment by AHaumer** [comment]
@casella You can push your changes / additions to my fork

File: Modelica/package.mo [hunk 1]
```diff
@@ -8280,7 +8283,7 @@ versionBuild=1,
 versionDate="2018-08-01",
 dateModified = "2018-08-01 12:00:00Z",
 revisionId="$Format:%h %s %ci$",
-uses(Complex(version="3.2.3"), ModelicaServices(version="3.2.3")),
+uses(                          ModelicaServices(version="3.2.3"), Complex(version="3.2.2")),
```
Comment (beutlich) [inline Modelica/package.mo]: This is probably an undesired side-effect.

**Comment by tobolar** [comment]
What about to add section headers such as
- `<h4>New libraries</h4>`
- `<h4>New components</h4>`
- etc.

**Comment by beutlich** [comment]
> I also note that we list these "breaking" changes at the end

What about the following mitigation? Instead of "errors that can lead to wrong simulation results" have "errors that can lead to unexpected or wrong simulation results"?

**Comment by beutlich** [comment]
> do a more complete rewrite

This simply was my initial comment on the CombiTimeTable issue (as one of the critical issue fixed). If you think it needs more clarification, so please go ahead.

**Pull request merged**